modules/document_repository/php/document_repository.class.inc

```diff
@@ -115,6 +124,8 @@ class NDB_Menu_Filter_Document_Repository extends NDB_Menu_Filter
                                'uploaded_by'   => 'v.uploaded_by',
                                'Date_uploaded' => 'v.Date_uploaded',
                               );
+
+        $this->_setFilterForm();
```

Comment:
This would not be required if the `_setFilterForm()` function name is changed to `_setupPage` instead of `_setupVariables()`

Reply:
I think I tried that and it didn't work.. (there's 2 tpl variables in setupVariables, and setupVariables gets called before setupFilterForm in NDB_Menu_Filter setup), but I can try again and double check.

The whole setup/setupPage is a bit of a mess for NDB_Menu_Filter, so it's difficult to see which way is the "right" way to do it..

Reply:
(I just tried again and, while the module works, I get errors printed in the console about the template variables defined in setupVariables not being defined printed to the screen.)

Reply:
~~It is because tpl_data should be set in `_setupPage` and not in `_setupVariables`~~
(edit) nevermind